cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -246,10 +246,7 @@ func canInstallAndRunAtOnce(mst *initramfsMountsState) (bool, error) {
 		return false, nil
 	}
 
-	// TODO: when install in initrd is finished and it is tested
-	// without fde hook, turn the return into...
-	// return true, nil
-	return kernelHasFdeSetup, nil
+	return true, nil
```

Comment:
are there not matching tests for this in the package?

Reply:
There wasn't one that tested the `false` case, I'll add one.